### ISSUE TITLE
Approximately how long should fine-tuning of the wav2vec small model take with 1h librispeech data?

### ISSUE THREAD

**Issue by chen7515** [facebookresearch/fairseq#4002]
I am fine-tuning the small wav2vec model with the 1 hour partition from the libri-light repo. I am using the base_1h.yaml configuration provided in this repo with some small changes to use 4 GPUs instead of 8:
update_freq=2,
distributed_world_size=4, 
max_tokens=3840000, 
freeze_finetune_updates=0.

I am not fine-tuning with a decoder. 
From reading other posted questions and issues, I understand that although the UER and raw WER will be close to or reach and stay at 100 early in the finetuning process, the fine tuned model should start producing <100 UER results once loss on the validation set drops below 4. 

I understand that the fine tuning setup configures for 13000 updates but I am observing that a single epoch which equates to approximately 2 - 3 updates is taking 15 seconds and it's increasing. So I am wondering if there's something wrong with my parallelization or utilization of the GPUs. How long approximately should fine-tuning take on 1h of Librispeech data? If you do not have approximate numbers for 4 GPUs, can you provide them for the 8 GPUs that were used in the 2020 paper?
